packages/graph/hash_graph/lib/graph/src/store/postgres/pool.rs

```diff
@@ -32,7 +32,7 @@ impl ErrorSink<Error> for ErrorLogger {
     }
 }
 
-#[allow(clippy::trait_duplication_in_bounds, reason = "false positive")]
+#[expect(clippy::trait_duplication_in_bounds, reason = "false positive")]
```

Comment:
Just more of a question I haven't asked about this. I see that we've replaced `allow` with `expect` a couple places. What's the reason for this? Does it have implications for how linting is done, or is `expect` scoped a different way than `allow`?

Reply:
With `expect` it's required that the link is triggered. This avoids us putting unneeded allow-attributes on statements.

Reply:
I.e. if clippy will be smart enough to resolve this correctly, this line will throw an error, so we can remove it again. This may only happen when bumping a toolchain or touching the file.

Reply:
Thanks both. Seems pretty powerful that it errors when the rule exception isn't used anymore :+1: